.github/workflows/serverless.yaml

```diff
@@ -0,0 +1,35 @@
+name: Serverless
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
```

Comment:
The step is called "Setup Python 3", but it looks like it installs Python 2.7.

Does ccm really need python 2?

Reply:
Previously I was testing with python version `3.x` on my fork, which caused some problems: https://github.com/Gor027/scylla-rust-driver/actions/runs/4591382447/jobs/8107506338

I am not sure what is the root cause of the problem, however, as `cqlsh` is compatible with Python 2.7, perhaps it is safer to prefer version 2.7 over 3.x

Reply:
Could you try again here with python 3? It looks like ccm tried to run some script from the downloaded "unified package" which required python 2. Maybe this problem was fixed in 5.1.6.

Reply:
OK, I see that it failed with the same error. In that case let's stick to python 2.